app/services/zendesk/pvb_api.rb

```diff
@@ -0,0 +1,40 @@
+module Zendesk
+  class PvbApi
```

Comment:
I'm not entirely sure about the naming here - what I am trying to do is to wrap the ZendeskAPI and abstract it away so we only expose what we need

Reply:
what about `Zendesk::PVBClient`?